blogReader.py

```diff
@@ -29,13 +31,13 @@ def __cacheArticlesID(self):
         self.__cachedArticlesID = []
         for dic in self.__cache:
             self.__cachedArticlesID.append(dic['_id'])
-        self.logger.info("'cacheArticlesID' updated")
 
     def recache(self):
         new_articles = self.checkUpdate()
         self.__cache = []
         self.__getArticles()
         self.__cacheArticlesID()
+        self.logger.info("'cacheArticlesID' updated")
```

Comment:
I'd prefer 
```suggestion
        self.logger.info("articles cache updated")
```
becausde the __cache array ios updated in factr, not the method you know